src/basic/signal-util.c

```diff
@@ -253,3 +253,38 @@ int signal_is_blocked(int sig) {
 
         return r;
 }
+
+int pop_pending_signal_internal(int sig, ...) {
+        sigset_t ss;
+        va_list ap;
+        int r;
+
+        if (sig < 0) /* Empty list? */
+                return -EINVAL;
+
+        if (sigemptyset(&ss) < 0)
+                return -errno;
+
+        /* Add first signal (if the signal is zero, we'll silently skip it, to make it easiert to build
```

Comment:
```suggestion
        /* Add first signal (if the signal is zero, we'll silently skip it, to make it easier to build
```

Reply:
Fixed directly in main, since this part was already merged in another PR.

e3709627e64ddf5d58aaf7c0bab08fda9fd7eb26